### DISCUSSION TITLE
v5.6.3

### DISCUSSION THREAD

**Discussion by j6s33m** [tsightler/ring-mqtt#739] [General Discussion]
FYI - It appears the latest release didn't publish to docker hub.

**Comment by tsightler** [comment]
Yes, I sometimes hold back the Docker images and addon updates for a few days if I know I'm not going to be around to respond to any issues because they impact so many more users.  They will be published when I'm back from a short trip.  In the meantime anybody that wants to use the new version can simply set the branch config option to "latest" and it will pull down the latest version from github during startup.
